components/emoji_picker/modules/emoji_skin_selector.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div data-qa="skin-selector">
+    <div
+      v-show="isOpen"
+      class="d-emoji-picker__skin-list"
+    >
+      <button
+        v-for="(skin, index) in skinList"
+        :key="skin.name"
+        :ref="`skinRef-${index}`"
+        :class="{
+          'selected': skinSelected?.skinCode === skin.skinCode,
+        }"
+        @click="selectSkin(skin)"
+        @keydown="event => handleKeyDown(event, skin, index)"
+      >
+        <img
+          class="d-icon d-icon--size-500"
+          :alt="skin.name"
+          :aria-label="skin.name"
+          :title="skin.name"
+          :src="`${cdnUrl + skin.unicode_output}.png`"
+        >
+      </button>
+    </div>
+    <div
+      v-show="!isOpen"
+      class="d-emoji-picker__skin-selected"
+    >
+      <!--      <dt-tooltip placement="top-end"> -->
+      <!--      {{ // skinSelectorButtonTooltipLabel }} -->
+      <!--        <template #anchor> -->
```

Comment:
This stuff supposed to be commented?

Reply:
Huh, ye, I definetly comentend that on purpose thinking that would be fix the storybook issues, lemme make it work again.

Reply:
k, now working, thanks 👍🏼